Source/Applications/SystemCenter/wwwroot/Scripts/TSX/SystemCenter/ExternalDB/ExternalDBTable.tsx

```diff
@@ -92,18 +92,25 @@ export default function ExternalDB(props: { ID: number, Tab: Tab }) {
             </div>
 
 
-            <hr />
-            <TabSelector CurrentTab={tab} SetTab={(t: Tab) => setTab(t)} Tabs={Tabs} />
-
-            <div className="tab-content" style={{ maxHeight: window.innerHeight - 235 }}>
-                <div className={"tab-pane " + (tab == "info" ? " active" : "fade")} id="info">
-                    <ExternalDBTableInfo Record={record} />
-                </div>
-                <div className={"tab-pane " + (tab == "fields" ? " active" : "fade")} id="fields">
-                    <ExternalDBTableFields TableName={record.TableName} ID={record.ID} />
-                </div>
-                <div className={"tab-pane " + (tab == "xda" ? " active" : "fade")} id="xda">
-                    <ExternalDBXdaFields ID={record.ID} />
+            <div className="row" style={{ flex: 1, overflow: 'hidden' }}>
```

Comment:
you could move the `TabSelector` up above this row (in it's own row, no Flex, no Height specified)  and resolve the `16px` comment that way